src/LinAlg/hiopMatrixSparseTriplet.cpp

```diff
@@ -706,11 +706,11 @@ void hiopMatrixSparseTriplet::print(FILE* file, const char* msg/*=NULL*/,
     if(NULL==msg) {
       if(numranks>1) {
         fprintf(file,
-                "matrix of size %lld %lld and nonzeros %lld, printing %d elems (on rank=%d)\n",
+                "matrix of size %d %d and nonzeros %d, printing %d elems (on rank=%d)\n",
                 m(), n(), numberOfNonzeros(), max_elems, myrank_);
       } else {
         fprintf(file,
-                "matrix of size %lld %lld and nonzeros %lld, printing %d elems\n",
+                "matrix of size %d %d and nonzeros %d, printing %d elems\n",
```

Comment:
Is there a reason not to use fstream here? The `<<` operator is overloaded for all int types.

Reply:
I think fstream is usually more appropriate, but if we do any real formatting with this, fstreams would be pretty painful and verbose